README.md

```diff
@@ -167,7 +167,7 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
   - [Wonder Women Tech Summit](https://wonderwomentech.com/)
 
-    Wonder Women Tech’s mission is to highlight, celebrate and educate women and the underrepresented in STEAM industries. We create conferences, events and workshops where we bring together changemakers, pioneers, innovators, and high-level speakers and attendees to explore ideas, discussions, and solutions for women and the underrepresented to succeed and thrive
+    Wonder Women Tech’s mission is to highlight, celebrate and educate women and the underrepresented in STEAM industries. 
```

Comment:
Thanks for making the changes, however, I do have a few things to add.

1. Sorry, I wasn't clear. I did not mean that you should remove the information, rather I meant you phrase it in a non-marketing way. Something like: 'Conferences, events and workshops where there high-level speakers. Attendees explore ideas, discussions, and solutions for women and the underrepresented.'

2. `STEAM` should be `STEM` I think.